native_client/ctcdecode/scorer.cpp

```diff
@@ -114,7 +124,7 @@ void Scorer::save_dictionary(const std::string& path) {
 }
 
 double Scorer::get_log_cond_prob(const std::vector<std::string>& words) {
-  double cond_prob;
+  double cond_prob = OOV_SCORE;
```

Comment:
I dont see this `OOV_SCORE`, where is it coming from?

Reply:
It's in `scorer.h`